wheelhouse.txt

```diff
@@ -0,0 +1,3 @@
+# Include python requirements here
+# Xenial & py35 support - https://github.com/juju-solutions/layer-basic/pull/197
+MarkupSafe<2.0.0
```

Comment:
We should at least define this for python < 3.6 and not for all versions. `MarkupSafe<2.0.0;python_version < '3.6'`